src/components/App/App.js

```diff
@@ -67,6 +67,14 @@ class App extends Component {
       favorites
     });
   };
+  // Mount Analytics Tracking code here
```

Comment:
Is it needed? It seems to already called here: https://github.com/Coding-Coach/find-a-mentor/pull/203/files#diff-528c3923d718a8860f5d8c05c3931c55R50

Reply:
Maybe he is refering to track a custom_event?

Reply:
He is using `create`. The custom event api looks different.

Reply:
@koolamusic ?